go/inst/instance_topology.go

```diff
@@ -2695,10 +2695,19 @@ func RelocateBelow(instanceKey, otherKey *InstanceKey) (*Instance, error) {
 	if err != nil || !found {
 		return instance, log.Errorf("Error reading %+v", *instanceKey)
 	}
+	// Relocation of group secondaries makes no sense, group secondaries, by definition, always replicate from the group
+	// primary
+	if instance.IsReplicationGroupSecondary() {
```

Comment:
I'm thinking the same logic needs to be applied in `MoveBelow`, `MoveBelowGTID`, `MakeCoMaster`, `Repoint`, `TakeMaster`, `MatchBelow`, `MatchUp` functions.

Reply:
Fixed